src/starcheck.pl

```diff
@@ -624,6 +624,7 @@ sub json_obsids{
     $obs{$obsid}->check_for_special_case_er();
     $obs{$obsid}->check_bright_perigee($radmon);
     $obs{$obsid}->count_good_stars();
+    $obs{$obsid}->check_big_box_stars();
```

Comment:
This isn't required, but as a reviewer, I'd prefer to have this or something like it in there to tide us over.